mitodl/smoot-design
# feat: Add Input, TextField components

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/6192

### Description (What does it do?)
Adds Input, Textfield components

Except for documentation tweaks, this is a direct port of Input and Textfield from MIT Learn

### How can this be tested?
1. Check out this branch, `yarn install`, `yarn start`
2. Visit http://localhost:6006/?path=/docs/smoot-design-input--docs It should make sense
3. Visit http://localhost:6006/?path=/docs/smoot-design-textfield--docs it should make sense 
4. In both cases, try out the storybook controls, too. Note: The stories aren't set up for the inputs to be typeable, but you can change the `value` prop via storybook controls.

### Additional Context
